packages/html-reporter/src/speedboard.tsx

```diff
@@ -15,36 +15,32 @@
  */
 
 import React from 'react';
-import { Filter } from './filter';
 import { LoadedReport } from './loadedReport';
 import { TestFileView } from './testFileView';
 import * as icons from './icons';
+import { TestCaseSummary } from './types';
 
-export function Speedboard({ filter, report }: { filter: Filter, report: LoadedReport}) {
+export function Speedboard({ report, tests }: { report: LoadedReport, tests: TestCaseSummary[] }) {
   return <>
-    <SlowestTests filter={filter} report={report} />
+    <SlowestTests report={report} tests={tests} />
   </>;
 }
 
-export function SlowestTests({ filter, report }: { filter: Filter, report: LoadedReport}) {
-  const [length, setLength] = React.useState(10);
-  const slowestTests = React.useMemo(() => {
-    const tests = report.json().files.flatMap(file => file.tests).filter(t => filter.matches(t));
-    return tests.sort((a, b) => b.duration - a.duration);
-  }, [report, filter]);
+export function SlowestTests({ report, tests }: { report: LoadedReport, tests: TestCaseSummary[] }) {
+  const [length, setLength] = React.useState(50);
   return <TestFileView
     file={{
       fileId: 'slowest',
       fileName: 'Slowest Tests',
-      tests: slowestTests.slice(0, length),
+      tests: tests.slice(0, length),
       stats: null as any,
     }}
     projectNames={report.json().projectNames}
     footer={
-      length < slowestTests.length
-        ? <button className='link-badge' onClick={() => setLength(l => l + 10)}>
+      length < tests.length
+        ? <button className='link-badge' onClick={() => setLength(l => l + 50)}>
           {icons.downArrow()}
-          Show more
+          Show 50 more
```

Comment:
This is lacking a hover state and the entire footer should be clickable (rather than just the text)